packages/fulfillment/src/services/fulfillment-provider.ts

```diff
@@ -78,8 +78,8 @@ export default class FulfillmentProviderService extends ModulesSdkUtils.internal
     return await provider.validateFulfillmentData(optionData, data, context)
   }
 
-  async validateOption(data: any) {
-    const provider = this.retrieveProviderRegistration(data.provider_id)
+  async validateOption(providerId: string, data: Record<string, unknown>) {
```

Comment:
Do we even need this in the custom service, or should we just put this logic in `fulfillment-module-service` instead?

Reply:
This is a service that communicate with the third party provider installed on the module, everything link to third party should be separated from the module service itself IMO, wdyt?